playwright.config.ts

```diff
@@ -5,7 +5,8 @@ const config: PlaywrightTestConfig = {
   updateSnapshots: 'none',
   expect: {
     toMatchSnapshot: {
-      threshold: 0.2,
+      threshold: 0.3,
+      maxDiffPixelRatio: 0.05,
```

Comment:
This is frustrating but not sure why the differences are occurring. After eyeballing the images in CI they look the same to the eye.